2015-03-01 16:36:59,776 INFO  [root][MainThread] Notification server connected
/root/pyramidenv/local/lib/python2.7/site-packages/SQLAlchemy-0.9.8-py2.7.egg/sqlalchemy/engine/default.py:585: SAWarning: Unicode type received non-unicodebind param value.
  processors[key](compiled_params[key])
2015-03-01 16:37:16,703 ERROR [waitress][Dummy-2] Exception when serving /api/notification
Traceback (most recent call last):
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/channel.py", line 337, in service
    task.service()
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/task.py", line 173, in service
    self.execute()
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/task.py", line 392, in execute
    app_iter = self.channel.server.application(env, start_response)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/paste/urlmap.py", line 203, in __call__
    return app(environ, start_response)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 242, in __call__
    response = self.invoke_subrequest(request, use_tweens=True)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 217, in invoke_subrequest
    response = handle_request(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_debugtoolbar-2.3-py2.7.egg/pyramid_debugtoolbar/toolbar.py", line 166, in toolbar_tween
    return handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/tweens.py", line 21, in excview_tween
    response = handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_tm-0.11-py2.7.egg/pyramid_tm/__init__.py", line 94, in tm_tween
    reraise(*exc_info)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_tm-0.11-py2.7.egg/pyramid_tm/__init__.py", line 75, in tm_tween
    response = handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 163, in handle_request
    response = view_callable(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 329, in attr_view
    return view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 305, in predicate_wrapper
    return view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 355, in rendered_view
    result = view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 477, in _class_requestonly_view
    response = getattr(inst, attr)()
  File "/root/pyramidenv/web/wondrous/utilities/general_utilities.py", line 141, in wrapper
    return func(self,*args,**kwargs)
  File "/root/pyramidenv/web/wondrous/views/ajax_main.py", line 431, in api_user_notification
    return NotificationManager.notification_json(user,**self.query_kwargs)
NameError: global name 'user' is not defined
Starting subprocess with file monitor
2015-03-01 16:38:57,485 INFO  [root][MainThread] Notification server connected
2015-03-01 16:39:12,575 ERROR [waitress][Dummy-2] Exception when serving /api/wall/new
Traceback (most recent call last):
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/channel.py", line 337, in service
    task.service()
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/task.py", line 173, in service
    self.execute()
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/task.py", line 392, in execute
    app_iter = self.channel.server.application(env, start_response)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/paste/urlmap.py", line 203, in __call__
    return app(environ, start_response)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 242, in __call__
    response = self.invoke_subrequest(request, use_tweens=True)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 217, in invoke_subrequest
    response = handle_request(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_debugtoolbar-2.3-py2.7.egg/pyramid_debugtoolbar/toolbar.py", line 166, in toolbar_tween
    return handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/tweens.py", line 21, in excview_tween
    response = handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_tm-0.11-py2.7.egg/pyramid_tm/__init__.py", line 94, in tm_tween
    reraise(*exc_info)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_tm-0.11-py2.7.egg/pyramid_tm/__init__.py", line 75, in tm_tween
    response = handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 163, in handle_request
    response = view_callable(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 329, in attr_view
    return view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 305, in predicate_wrapper
    return view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 355, in rendered_view
    result = view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 477, in _class_requestonly_view
    response = getattr(inst, attr)()
  File "/root/pyramidenv/web/wondrous/utilities/general_utilities.py", line 141, in wrapper
    return func(self,*args,**kwargs)
  File "/root/pyramidenv/web/wondrous/views/ajax_main.py", line 356, in api_new_post
    retval = PostManager.post_json(**query_kwargs)
TypeError: post_json() takes at least 4 arguments (2 given)
2015-03-01 16:39:31,173 ERROR [waitress][Dummy-4] Exception when serving /api/wall/new
Traceback (most recent call last):
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/channel.py", line 337, in service
    task.service()
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/task.py", line 173, in service
    self.execute()
  File "/root/pyramidenv/local/lib/python2.7/site-packages/waitress-0.8.9-py2.7.egg/waitress/task.py", line 392, in execute
    app_iter = self.channel.server.application(env, start_response)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/paste/urlmap.py", line 203, in __call__
    return app(environ, start_response)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 242, in __call__
    response = self.invoke_subrequest(request, use_tweens=True)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 217, in invoke_subrequest
    response = handle_request(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_debugtoolbar-2.3-py2.7.egg/pyramid_debugtoolbar/toolbar.py", line 166, in toolbar_tween
    return handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/tweens.py", line 21, in excview_tween
    response = handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_tm-0.11-py2.7.egg/pyramid_tm/__init__.py", line 94, in tm_tween
    reraise(*exc_info)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid_tm-0.11-py2.7.egg/pyramid_tm/__init__.py", line 75, in tm_tween
    response = handler(request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/router.py", line 163, in handle_request
    response = view_callable(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 329, in attr_view
    return view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 305, in predicate_wrapper
    return view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 355, in rendered_view
    result = view(context, request)
  File "/root/pyramidenv/local/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/config/views.py", line 477, in _class_requestonly_view
    response = getattr(inst, attr)()
  File "/root/pyramidenv/web/wondrous/utilities/general_utilities.py", line 141, in wrapper
    return func(self,*args,**kwargs)
  File "/root/pyramidenv/web/wondrous/views/ajax_main.py", line 356, in api_new_post
    retval = PostManager.post_json(**query_kwargs)
TypeError: post_json() takes at least 4 arguments (2 given)
